src/js/jsx/search/SearchBar.jsx

```diff
@@ -229,10 +227,23 @@ define(function (require, exports, module) {
 
                         return toRemove ? toRemove.replace(" ", "") : idWord;
                     }, this).join("").toLowerCase(),
+                    partialMatch,
```

Comment:
This variable is used by the subsequent variable declarations, therefore must be present here. sorry for the ugly.